src/meta/processors/BaseProcessor.inl

```diff
@@ -93,6 +93,44 @@ void BaseProcessor<RESP>::doMultiRemove(std::vector<std::string> keys) {
     this->onFinished();
 }
 
+template<typename RESP>
+kvstore::ResultCode BaseProcessor<RESP>::multiRemove(std::vector<std::string> keys) {
+    folly::Baton<true, std::atomic> baton;
+    kvstore::ResultCode retCode;
+    kvstore_->asyncMultiRemove(kDefaultSpaceId,
+                               kDefaultPartId,
+                               std::move(keys),
+                               [this, &baton, &retCode] (kvstore::ResultCode code) {
+        this->handleErrorCode(MetaCommon::to(code));
+        retCode = code;
+        baton.post();
+    });
+    baton.wait();
+    return retCode;
+}
+
+template<typename RESP>
+kvstore::ResultCode BaseProcessor<RESP>::multiRemoveAndUpdate(std::vector<std::string> keys) {
```

Comment:
Useless interface. and the `LastUpdateTimeMan::update` has beed added in `doSyncPutAndUpdate` and `doSyncMultiRemoveAndUpdate`

Reply:
Not same, I don't finish the processor here.